share/dictionary/dhcpv4/dictionary.rfc2485

```diff
@@ -0,0 +1,14 @@
+# -*- text -*-
+# Copyright (C) 2022 The FreeRADIUS Server project and contributors
+# This work is licensed under CC-BY version 4.0 https://creativecommons.org/licenses/by/4.0
+# Version $Id$
+##############################################################################
+#
+#  RFC2485 - DHCP Option for The Open Group's User Authentication Protocol
+#
+#  $Id$
+#
+##############################################################################
+
+# Open Group's User Authentication
+ATTRIBUTE	User-Auth				98	octets
```

Comment:
This is `string`, a list of URLs separated by space.

Reply:
Corrected.